### PR TITLE
Add a way to retrieve the raw storage

### PR DESCRIPTION
**Pull request by WalshyDev**
## Description

Provide a way to access BlockStorage's `storage`. This adds a way to access all the blocks loaded in the server. It may be worth making this return an immutable map so that people don't modify this. Let me know your opinion.

This is currently untested (though if it compiles, can't see why it wouldn't work) and I made this on my phone sooo compilation may fail. Not added tests or docs either due to phone (also this class really needs to be redone anyway)

## Changes

* Added BlockStorage#getStorage
* Added BlockStorage.getStorage(World)

## Related Issues

N/A

## Checklist

- [ ] I have fully tested the proposed changes and promise that they will not break everything into chaos.
- [ ] I have also tested the proposed changes in combination with various popular addons and can confirm my changes do not break them.
- [x] I followed the existing code standards and didn't mess up the formatting.
- [ ] I did my best to add documentation to any public classes or methods I added.
- [x] I have added `Nonnull` and `Nullable` annotations to my methods to indicate their behaviour for null values
- [ ] I added sufficient Unit Tests to cover my code.

### CODE REVIEW

**Review by Sfiguz7 — DISMISSED** [review]
No tests and phone coding >:( But LGTM

File: src/main/java/me/mrCookieSlime/Slimefun/api/BlockStorage.java [hunk 1]
```diff
@@ -360,6 +362,23 @@ public static void saveChunks() {
         }
     }
 
+    @Nonnull
+    public Map<Location, Config> getRawStorage() {
+        return this.storage;
```
Comment (TheBusyBiscuit) [inline src/main/java/me/mrCookieSlime/Slimefun/api/BlockStorage.java:367]: Like you presumed, I'd recommend making this immutable.

**Review by TheBusyBiscuit — CHANGES_REQUESTED** [review]
I don't really see much of a point in this, especially since we will be tearing down this class at some point anyway but lgtm, just... immutable pls

File: src/main/java/me/mrCookieSlime/Slimefun/api/BlockStorage.java [hunk 2]
```diff
@@ -360,6 +363,36 @@ public static void saveChunks() {
...
+     * @param world The world of which to fetch the data from.
+     * @return An {@link ImmutableMap} of the raw data or null if the world isn't registered.
+     */
+    @Nullable
+    public static Map<Location, Config> getRawStorage(@Nonnull World world) {
+        Validate.notNull(world, "World cannot be null!");
+
+        BlockStorage storage = getStorage(world);
+        if (storage != null) {
+            return (storage.getRawStorage();
```
Comment (Sfiguz7) [inline src/main/java/me/mrCookieSlime/Slimefun/api/BlockStorage.java]:
```suggestion
            return storage.getRawStorage();
```

**Review by Sfiguz7 — APPROVED** [review]

File: src/main/java/me/mrCookieSlime/Slimefun/api/BlockStorage.java (hunk 2 above)
Comment (Sfiguz7) [reply src/main/java/me/mrCookieSlime/Slimefun/api/BlockStorage.java]: Didn't mean to approve since I suggested the above, eh

**Pull request merged**